openquake/hazardlib/source/multi_fault.py

```diff
@@ -88,7 +89,7 @@ def todict(self):
         """
         ridxs = []
         for rupture_idxs in self.rupture_idxs:
-            ridxs.append(' '.join(rupture_idxs))
+            ridxs.append(' '.join(f'{rupture_idxs}'))
```

Comment:
I do not understand what it is the difference with the respect to the original version (?)

Reply:
I guess we should use `ridxs.append(' '.join(map(str, rupture_idxs)))`